odxtools/cli/browse.py

```diff
@@ -101,12 +115,12 @@ def encode_message_interactively(sub_service, ask_user_confirmation=False):
     for param_or_dict in param_dict.values():
         if isinstance(param_or_dict, dict):
             for param in param_or_dict.values():
-                if param.is_settable:
+                if not isinstance(param_or_dict, dict) and param.is_settable:
```

Comment:
why are you testing on dict?

Reply:
param is of type `ParameterValue`, in this particular location we want to ask the user for a value since querying structures is not (yet?) implemented...